### PR TITLE
Move user.css to data folder

### PR DESCRIPTION
**Pull request by ArthurHoaro**
Reasons:

  - easier to update (just keep data folder)
  - no git conflicts

### CODE REVIEW

File: application/Updater.php [hunk 1]
```diff
@@ -308,6 +308,22 @@ public function updateMethodDefaultTheme()
...
+    /**
+     * Move the file to inc/user.css to data/user.css.
+     *
+     * Note: Due to hardcoded paths, it's not unit testable. But one line of code should be fine.
+     *
+     * @return bool true if the update is successful, false otherwise.
+     */
+    public function updateMethodMoveUserCss()
+    {
+        if (! is_file('inc/user.css')) {
```
Comment (virtualtam) [inline application/Updater.php]: Note: `is_file()` seems to resolve symlinks, i.e. check if the path is a symbolic link pointing to an existing file

For clarity, we should also check if this location corresponds to a symlink using [is_link()](https://secure.php.net/manual/en/function.is-link.php)

Comment (ArthurHoaro) [inline application/Updater.php]: I just tried, using `rename()` on a symlink works just fine.

Comment (virtualtam) [inline application/Updater.php]: Yup, it's sad the documentation doesn't mention it :(

Comment (ArthurHoaro) [inline application/Updater.php]: I agree. I find the doc great and understandable, but often lacks specific details.

**Pull request merged**